.jshintrc

```diff
@@ -13,8 +13,7 @@
   "unused": "vars",
 
   "node": true,
-  
+
   "trailing": true,
-  "curly": true,
-  "indent": 2
```

Comment:
I removed "indent" for two reasons: one, this PR leaves indentation intact to make it easier to see where the changes lie and to temporarily maintain git history (if we want to make another commit to fix/normalize indentation before merging we can do that, or we could do it as a separate PR later); and two, JSHint has deprecated style-related properties like Indent. We should add [JSCS](http://jscs.info/) to take over the responsibility of checking for stylistic consistency.